fiftyone/core/threed/object_3d.py

```
@@ -223,6 +234,186 @@ def clear(self) -> None:
+                removed.add(child)
+            return True
+
+        self.find_and_execute(self, predicate, on_match)
+
+        if not removed:
+            raise ValueError(
+                f"Objects with name '{name}' not found in scene graph"
+            )
+
+    def remove_by_uuid(self, uuid: str) -> None:
+        """Remove the object with the given UUID from the scene graph recursively.
```

Pylint (4.0.3)

[convention] Line too long (82/79)

(C0301)